packages/navbar/src/NavLinkContent.js

```diff
@@ -16,7 +17,17 @@ export default function (props) {
     iconComponent = <span key='no-selected' className={iconClassName}><i className={`w-100 h-100 ${noneIcon}`} /></span>
     subtitle = '(None)'
   } else {
-    iconComponent = <span key={key} className={iconClassName}><i className={`w-100 h-100 ${icon}`} /></span>
+    if (title == 'Network' && selected) {
+      let ethereummain = ['homesteadmain']
+      let ethereumtest = ['ropstentest', 'rinkebytest', 'goerlitest', 'kovantest']
+      let iconUrl = selected.split(' ').join('').toLowerCase()
+      iconUrl = iconUrl.slice(0, iconUrl.length - 3)
```

Comment:
I believe `iconFlag` will better than `iconUrl` and why not use the `group` prop from the `networks`?
https://github.com/ObsidianLabs/eth-components/blob/master/packages/eth-sdk/src/networks.js#L6